<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Process Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>New processes</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="startButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="numOfProcessesLineEdit">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Amount of processes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="processesArgumentsLineEdit">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>size x size</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>State</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="resumeButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Resume</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="suspendButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Suspend</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="terminateButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Terminate</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Priority</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="setPriorityButton">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Change priority</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="proirityComboBox">
               <property name="currentText">
                <string>BELOW_NORMAL_PRIORITY_CLASS</string>
               </property>
               <item>
                <property name="text">
                 <string>IDLE_PRIORITY_CLASS</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>BELOW_NORMAL_PRIORITY_CLASS</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>NORMAL_PRIORITY_CLASS</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ABOVE_NORMAL_PRIORITY_CLASS</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>HIGH_PRIORITY_CLASS</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>REALTIME_PRIORITY_CLASS</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="processesTable">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>0</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>24</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Priority</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ProcessTime</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>KernelTime</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>UserTime</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
